test/ng/compileSpec.js

```diff
@@ -2762,6 +2762,23 @@ describe('$compile', function() {
     }));
 
 
+    it('should handle consecutive text elements as a single text element', inject(function($rootScope, $compile) {
+      window.addMutationObserver = function() {
```

Comment:
But if the function isn't called, then there is no MutationObserver added right ?
How does the function help if we don't call it ?
And we does the function need to be assigned on window ?